src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_RANDOM_VALUE = 100;
+    public static final int NUMBER_FIGURES = 5;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = new Random().nextInt(NUMBER_FIGURES);
```

Comment:
```suggestion
        int index = random.nextInt(NUMBER_FIGURES);
```
please be careful when refactoring your code